mobile/examples/image_classification/android/prepare_models.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+import argparse
+import onnx.hub
+import pathlib
+import shutil
+import subprocess
+import sys
+import tempfile
+import urllib.request
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Prepares model files used by the image_classification example.")
+    default_format = "onnx"
+    parser.add_argument(
+        "--format",
+        choices=["onnx", "ort"],
+        default=default_format,
+        help=f"Model format to generate. Default: {default_format}",
+    )
+    parser.add_argument(
+        "--output_dir",
+        type=pathlib.Path,
+        required=True,
+        help="Path to output directory.",
+    )
+    return parser.parse_args()
+
+
+def download_file(url: str, output_path: pathlib.Path):
+    with urllib.request.urlopen(url) as response:
+        assert response.status == 200
+        with open(output_path, mode="wb") as file:
+            file.write(response.read())
+
+
+def download_onnx_model(model_name: str, output_path: pathlib.Path):
+    model = onnx.hub.load(model_name)
+    onnx.save(model, str(output_path))
+
+
+def main():
+    args = parse_args()
+
+    args.output_dir.mkdir(parents=True, exist_ok=True)
+
+    # download classes file
+    download_file(
+        "https://raw.githubusercontent.com/pytorch/hub/master/imagenet_classes.txt",
+        args.output_dir / "imagenet_classes.txt",
+    )
+
+    # prepare models in temporary directory, then copy them to output directory
+    with tempfile.TemporaryDirectory(dir=args.output_dir) as temp_dir:
+        temp_dir = pathlib.Path(temp_dir)
+
+        for model_name, model_path_stem in [
+            ("MobileNet v2-1.0-int8", "mobilenetv2_int8"),
```

Comment:
nit: Is it okay to use int8 mobilenet model? just noticed previously the model was an uint8 one.

Reply:
it seems to work. I'm not sure how much of a difference it makes. @yufenglee any idea?
the ONNX model zoo only has an int8 quantized version.

Reply:
I would recommend int8 model for mobile. Int8 will be faster than uint8 on arm64 without dot product.